modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestWithoutAccessToken.siddhi

```diff
@@ -0,0 +1,73 @@
+@App:name("PublishHttpOAuthRequest")
+@App:description("Send an OAuth events via HTTP transport without access token")
+/*
+Purpose:
+	This application demonstrates how to configure WSO2 Stream Processor to send an OAuth events via HTTP transport
+	without access token.
```

Comment:
Here also remove 'OAuth events' part (we are sending HTTP events to an OAuth-protected endpoint).